velox/connectors/hive/PartitionIdGenerator.cpp

```diff
@@ -66,7 +68,21 @@ void PartitionIdGenerator::run(
   const auto numRows = input->size();
   result.resize(numRows);
 
-  // TODO Check that there are no nulls in the partition keys.
+  if (rejectNullPartitionKeys_) {
```

Comment:
Wondering if this is the right place for this check. Perhaps, a better place would be in the caller.